### DISCUSSION TITLE
Scheduler triggers events 5 hours earlier, which also happens to match my GMT offset (-5:00)

### DISCUSSION THREAD

**Discussion by si-hb** [jensrossbach/node-red-contrib-chronos#75] [Q&A] [answered]
I can't figure out what I'm doing wrong. The node-red debugger is time stamping events with the correct time, my server has the correct time, the scheduler node is reporting 'next event' time correctly, yet the schedule node is triggering events 5 hours earlier than when they're supposed to. (if I have an event set for 9:00 pm, the event actually fires at 4:00pm)

Any help is immensely appreciated.

Brilliant work on this node collection btw, thank you!

**Comment by jensrossbach** [comment, marked as answer]
Hi, in first place it looks like a problem with the configuration of the time zone on your Node-RED server. But as you stated that you ensured that the configuration is correct, I have no clue what could be the problem. I never observed such behavior on my side.

**Reply by diedder** [reply]
Hi,
I have exactly the same issue. The difference is 1h, which is the the difference from zulu to local for germany. Other nodes as ui-clock or cron-nodes are working with the correct local time. The scheduler node seems to use zulu time for the events.
could yo pls check this and let me know if you need more infos - thank you!

**Reply by jensrossbach** [reply]
Scheduler node users Moment.js for all datetime related calculations and I believe that Moment.js handles the time zone correctly. Anyway to further analyze this, can you please reproduce the issue with logs on debug or better trace level activated? Then please create a bug issue in the issue tracker and provide the traces, thanks.

**Reply by si-hb** [reply]
@jensrossbach From your reply, I went back and checked the time-zone config of Node-red. It was set to 'ET' (Eastern Time) here in Canada, I suppose it was set that way automatically when it was first deployed to my Synology DS. It is the correct timezone for Canada, however to be sure, I updated it to 'Canada/Eastern' and that seems to have fixed the problem, thank you.

**Reply by diedder** [reply]
@jensrossbach I could also fix it. In my virtual configuration there has to be set an special addional variable to set the timezone. It seems that moments uses exactly this one while node red is working in the correct timezone without it. Thank you anyway!
